packages/modules/caching/src/types/index.ts

```diff
@@ -32,6 +57,14 @@ export type CachingModuleOptions = Partial<ModuleServiceInitializeOptions> & {
    * Time to keep data in cache (in seconds)
    */
   ttl?: number
+
+  /**
+   * Enable and configure the built in memory cache
+   */
+  in_memory?: MemoryCacheModuleOptions & {
+    enable?: boolean
+  }
```

Comment:
thought(non-blocking): should we add the `@private` on this property until we have a clearer picture of how/if we want to use it?